be/src/exec/hash_table.hpp

```diff
@@ -72,6 +70,7 @@ inline HashTable::Bucket* HashTable::next_bucket(int64_t* bucket_idx) {
     return NULL;
 }
 
+// TODO:
```

Comment:
todo what here

Reply:
Sorry, this is a useless comment. I'll fix it